lib/ledger-node-operations.js

```diff
@@ -23,11 +51,11 @@ module.exports = class LedgerNodeOperations {
    * operation - the operation to perform.
    * options - a set of options used when adding the operation.
    * callback(err) - the callback to call when finished.
-   *   err - An Error if an error occurred, null otherwise.
+   *   err - A BedrockError if an error occurred, null otherwise.
    */
   add({operation}, callback) {
     if(!operation) {
-      throw new TypeError('`operation` is required.');
+      throw new BedrockError('`operation` is required.');
```

Comment:
Should be `return callback(new BedrockError(...))`.  Does this error need a type?

Reply:
Maybe align the add() and validate() checks to be similar?

Reply:
We should do `throw new TypeError` to be consistent in other places. And this will get handled more cleanly when we switch to promises.